MarmoUI-Chrome/manifest.json

```diff
@@ -2,17 +2,18 @@
   "manifest_version": 2,
 
   "name": "MarmoUI",
-  "description": "Marmoset Improved! Better UI and functionality",
+  "description": "test Marmoset Improved! Better UI and functionality",
   "version": "1.5",
   "icons": {
     "16": "image/icon16.png",
     "48": "image/icon48.png",
-    "128": "image/icon128.png" 
+    "128": "image/icon128.png"
   },
-  
+
   "content_scripts": [
     {
-      "matches": ["https://marmoset.student.cs.uwaterloo.ca/*"],
+      "matches": ["https://*.uwaterloo.ca/*"],
```

Comment:
Does this show up in the extension as "Can access anything in *.uwaterloo.ca"? Some students might wonder why that is...

Also would it work if we just put "https://marmoset*.uwaterloo.ca/*"?